MarkBind/markbind
# Add relative links to the convert template file

**What is the purpose of this pull request?**


Fixes #1725

**Overview of changes:**
This PR directly address the issue raised in #1725.

I fix it by adding the {{baseUrl}} into the template nunjucks file. 

**Anything you'd like to highlight / discuss:**


**Testing instructions:**

Run `markbind init -c` on a test repo and see that the {{baseUrl}} is indeed there.

![image](https://user-images.githubusercontent.com/19247856/149976767-b7bd6087-3085-491b-b9cf-6cabb0fa8202.png)


**Proposed commit message: (wrap lines at 72 characters)**

Add relative links to template file

---

**Checklist:** :ballot_box_with_check:
